va/va_fei_h264.h

```diff
@@ -318,6 +318,115 @@ typedef struct _VAEncFEIDistortionH264 {
     uint32_t    reserved1[2];
 } VAEncFEIDistortionH264;    // 48 bytes
 
+/** \brief Motion Vector and Statistics frame level controls.
+ * VAStatsStatisticsParameterBufferType for H264 16x16 block
+ **/
+typedef struct _VAStatsStatisticsParameterH264
+{
+   VAStatsStatisticsParameter stats_params;
+
+    uint32_t    frame_qp                    : 8;
+    /** \brief maximum number of Search Units, valid range is [1, 63] */
+    uint32_t    len_sp                      : 8;
+    /** \brief motion search method definition
+     * 0: default value, diamond search
+     * 1: full search
+     * 2: diamond search
+     **/
+    uint32_t    search_path                 : 8;
+    uint32_t    reserved0                   : 8;
+
+    uint32_t    sub_mb_part_mask            : 7;
+    uint32_t    sub_pel_mode                : 2;
+    uint32_t    inter_sad                   : 2;
+    uint32_t    intra_sad                   : 2;
```

Comment:
I think we should add an explanation for all these fields, like "01 indicate half and 11 indicate quarter pixel interpolation" right?

Reply:
Yes, we should have comments for it like below,
00b: integer mode searching
01b: half-pel mode searching
10b: reserved
11b: quarter-pel mode searching